### DISCUSSION TITLE
討厭馬賽克？Google 表示你可以「修復」它了

### DISCUSSION THREAD

**Discussion by 20010906** [1101-GS4719-C/Python-Community-GS4719-C#27] [General]
如標題，Google AI 團隊利用兩種演算法，提高低解析度照片的畫素，先用高斯雜訊演算法，破壞樣本，組成雪花圖，再利用神經網路演算法，逆轉高斯雜訊的破壞，並合成新圖片。我覺得這篇文章很有趣，且與生活相關，與大家分享~
[文章連結](https://technews.tw/2021/09/20/google-pixelization-repair/)

**Comment by chupochun** [comment]
搞不好以後可用在修復不清晰的照片上

**Comment by pekoranoinu** [comment]
把馬賽克給修復掉一直是我的夢想阿!

**Comment by Ben6uo** [comment]
但可以把馬賽克給修復掉是不是會更容易在網絡肉搜別人?

**Comment by ghost** [comment]
不過諸如此類的東西，往往都在倫理道德邊緣的模糊地帶 阿哈哈...

**Comment by bloodtrail0511** [comment]
感覺這樣的修復方式，還需要配置一個強大的AI算法，不然修復後也會有失真的可能

**Comment by xkllkx** [comment]
在網路世界真的沒有私密性可言了

**Comment by d26488399** [comment]
我認為這項技術拿來破案可能是個不錯的選擇。

**Comment by PoooYen** [comment]
我有很多個大膽的想法哈哈哈，(調正google map的畫質之類的

**Comment by awkward-willy** [comment]
這項技術應用在模糊照片的影像清晰化上
想必能帶來很多好處
但相對來說，若運用在馬賽克等本就用來遮擋的東西上
則會衍伸出許多隱私相關的問題
可見技術與道德之間的拿捏也是相當重要的

**Comment by jason79461385** [comment]
之前有用過類似的方法是透過周遭方格的顏色來填補空缺的地方，以此來讓整個照片看起來更正常些

**Comment by sys511613** [comment]
這個技術可以在生活上幫助很多，雖然還原過後的東西還是會跟原圖有些誤差，但是透過這個技術可以讓我們看到圖片中更多的細節，最重要的是不會有看到解析度很差的圖的阿雜感

**Comment by HaoFish123** [comment]
感謝分享，但就怕被有心人亂使用。

**Comment by alexthebigpp** [comment]
這樣遲早又會出現比現在馬賽克更厲害的馬賽克技術。

**Comment by yenwei12138** [comment]
感覺就好像把拼圖弄亂在重拚起來一樣

**Comment by Lynn12378** [comment]
感覺可以用在修復老照片

**Comment by etin2468** [comment]
如果只是拿來修復照片是好事，但如果是拿來破壞隱私就不太妙了

**Comment by Hchejxnc** [comment]
一些老的照片解析度沒有很高，用它或許解決問題了~

**Comment by Evan0979** [comment]
比起修復馬賽克,我更想拿來修復那些古早的老照片,感覺對考古和歷史的研究有很大的幫助

**Comment by manipulateed** [comment]
這項技術或許能彌補早期解析度的不足，使人們可以重新認識和看清過往影視的風采。

**Comment by williamyen0623** [comment]
太酷的技術了吧！但如果被不當使用那可就不好了

**Comment by evanchou1204** [comment]
可以用來解決照片傳來傳去解析度降低的問題，只是不知道會不會失真，修復成不是原來的樣子就好笑了。

**Comment by Pietra1226** [comment]
這樣有的犯罪受害者圖像或是其他不能公開的圖像就有可能會被解碼，這樣就牽涉到隱私的問題了

**Comment by dabbie109303501** [comment]
這樣修復老舊照片或影片就很方便了

**Comment by Doreen331** [comment]
會用到馬賽克就是因為想碼掉不想讓別人看到的東西，若這項技術大家都能使用，會造成很大的道德爭議，雖然能夠被利用來修復照片，但若修復一些例如車禍現場的血腥照片就不太好，也很難管控被修復後的照片的流向

**Comment by Kotori0501** [comment]
感謝分享
平時有在收集動漫相片和辨識、分析圖像，但有些照片模糊不清楚，所以有這項技術便可解決問題

**Comment by Chiway1229** [comment]
那這樣之前遇到的那些 有馬賽克的照片都能夠修復了
好耶

**Comment by yuxinlin93** [comment]
如果能夠修復馬賽克，感覺會被大量的不當使用也

**Comment by ruthchen1020** [comment]
之前看過的資料是說不行耶，只是用機器學習區去推測原本的顏色而已。

**Comment by ghost** [comment]
我覺得在修復照片解析度上，這會是很棒的發明，因為每台設備不一定都是具備高解析度，如果這項技術成熟，解析度就不在被設備給約束住，對於消費者來說是一大福音；然，對於能否破解馬賽克上，我對此抱持懷疑，馬賽克的目的就是為了保障該部分的隱私不被透漏，若能夠破解，那勢必會出現新的馬賽克生成算法來實現 保障隱私 這個目的，所以這就會產生一個問題 出題困難還是解題困難，如果出題困難的話，那勢必所有的改進的算法都有辦法被解決；但如果是解題難的話，那只要算法不停更新，就永遠也無法解開。(好像扯遠了)

**Comment by StevenWanglolz** [comment]
利用這個科技我們就可以修復很多以前模糊不清的照片了! 像是一些監視器拍到的鬼、飛碟、和雪怪的照片了!!! 太讚了!

**Comment by zhiron544** [comment]
我覺得Google AI 團隊的這個技術很有趣，可以讓我們看到更清晰的照片，也可以修復一些模糊或馬賽克的圖像。我想知道這個技術的原理和效果，以及它有什麼優點和缺點。

**Comment by Alex11392** [comment]
感覺如果這個功能可以很真實的還原馬賽克，可能未來會有新的馬賽克方式

**Comment by poiu6fi** [comment]
說不定馬賽克被破解後，之後會有新的方式去遮蓋那些圖片，一山還有一山高

**Comment by cassy0201** [comment]
我覺得如果馬賽克可以輕易修掉可能會造成個資和隱私權的問題，可能還有機會被有心人士利用

**Comment by shrshu38** [comment]
這技術雖然很酷，但也可能引發隱私和道德問題。比如本來打了馬賽克的畫面，被「修復」後就失去保護效果，可能被拿來惡意還原個資或侵犯他人隱私。未來如果濫用，會讓影像保護變得更加困難。
